examples/bigcommerce/components/cart.tsx

```diff
@@ -202,7 +212,8 @@ export function ProductAddToCartButton({ className }: ProductAddToCartButtonProp
   const [quantity, setQuantity] = useState(1)
   const [addToCartState, setAddToCartState] = useState<AddToCartState>('initial')
   const { addItem } = useCart()
-  const product = useProduct()
+  const product = useProductFromPath()
```

Comment:
this hook was renamed to make way for `getProduct(id: number)`

Reply:
I'm not sure I understand this comment. How does renaming `useProduct` to `useProductFromPath` make way for a function `getProduct` that's not a hook?

Reply:
Sorry I meant `getProduct(id: number)`

Reply:
You just wrote exactly what you wrote in your original comment. Do you mean `useProduct(id: number)`?